lib/classes/PluginManager.js

```diff
@@ -275,7 +286,7 @@ class PluginManager {
           }
           return event;
         });
-        this.commands[key] = _.merge({}, this.commands[key], command);
+        this.commands[key] = _.merge({}, this.commands[key], command, options);
```

Comment:
It'll be more future proof, to specifically pass `{ isExternal: options.isExternal }` (by design `options` are `loadCommands` operation options, and not command extension, and here we treat it as if it's command extension)